01-node-tutorial/answers/11-fs-async.js

```diff
@@ -0,0 +1,12 @@
+const { writeFile } = require("fs");
+console.log("at start");
+
+writeFile("./temporary/fileB.txt", "This is line 1\n", (err, result) => {
+  console.log("at point 1");
+  if (err) {
+    console.log("This error happened: ", err);
+  } else {
+    console.log("Done with this task")
```

Comment:
You got the command correct, however, there needs to be more writeFile calls in the callback